dev_scripts/env.py

```diff
@@ -575,8 +621,26 @@ def run(
         (dist_state / ".bash_history").touch(exist_ok=True)
         self.runtime_run(*run_cmd)
 
-    def build_dev(self, show_dockerfile=DEFAULT_SHOW_DOCKERFILE):
+    def pull_image_from_registry(self, image):
+        process = subprocess.run(self.runtime_cmd + ["pull", IMAGES_REGISTRY + image])
+        return process.returncode == 0
+
+    def push_image_to_registry(self, image):
+        process = subprocess.run(
+            self.runtime_cmd + ["push", image, IMAGES_REGISTRY + image]
+        )
+        return process.returncode == 0
```

Comment:
I'd probably add `check=True` here, and capture the stderr, to learn about errors.

Reply:
Thanks, fixed in 0ae59f3b6d63b55cd8fc107188ada6df5f250edb